### DISCUSSION TITLE
Dose unit in Gy per second

### DISCUSSION THREAD

**Discussion by physicsmed** [nrc-cnrc/EGSnrc#1155] [Questions]
Hello Scientists, 
Many research have mentioned that the average dose rate in unit of Gy/s or Gy/min. based on my knowledge, the 3ddose file stores dose in unit of Gy/incident particle. I am wondering if there is any way to determine the dose in unit of Gy/s?

**Comment by rtownson** [comment]
If you are modelling a linac, see this paper on calibrating your MC simulation for absolute dose:

https://pubmed.ncbi.nlm.nih.gov/16177516/

Note that backscatter corrections may be helpful:

https://pubmed.ncbi.nlm.nih.gov/24487824/
